src/DynamoCore/Search/SearchDictionary.cs

```diff
@@ -12,7 +12,7 @@ namespace Dynamo.Search
     /// </summary>
     public class SearchDictionary<V>
     {
-        private readonly Dictionary<V, Dictionary<string, double>> entryDictionary =
+        protected readonly Dictionary<V, Dictionary<string, double>> entryDictionary =
```

Comment:
I have to change from `private` to `protected`, because I need to use it in `SearchLibrary`.